test/test_ensembling_pipelines_generation.py

```diff
@@ -29,7 +30,9 @@ def tearDownClass(cls):
 
     def test_experimenter_can_generate_basic(self):
         # generate the pipelines
-        generated_pipelines = self.experiment.generate_k_ensembles(k_ensembles=3, p_preprocessors=1,
+        generated_pipelines = self.experiment._generate_k_ensembles(k_ensembles=3, n_preprocessors=1,
+                                                                        preprocessors=preprocessors,
+                                                                        models=models,
```

Comment:
`_generate_k_ensembles` now needs to be passed `preprocessors` and `models` because it no longer has direct access to the attributes of the `Experimenter` class.